telegram/ext/_picklepersistence.py

```diff
@@ -28,11 +28,50 @@
     cast,
 )
 
+from telegram import Bot
 from telegram._utils.types import FilePathInput
+from telegram._utils.warnings import warn
 from telegram.ext import BasePersistence, PersistenceInput
 from telegram.ext._contexttypes import ContextTypes
 from telegram.ext._utils.types import UD, CD, BD, ConversationDict, CDCData
 
+_REPLACED_KNOWN_BOT = "known_bot"
```

Comment:
should be something more unique in a final version :D

Reply:
should do the job :) the content doesn't really matter - we could also just use a uuid4. we'll just have to make sure that it's unlikely for someone to actually use that string elsewhere …